CONTRIBUTING.md

```diff
@@ -49,21 +49,16 @@ Merging Pull Requests (for project contributors with write access)
 - Install [Maven](http://maven.apache.org/download.cgi)
   - add `MAVEN_HOME` to environment variables
 
```

Comment:
The note is still valuable. Unless it is listed elsewhere in this document.

Reply:
The long path issue has been solved. I can add it back in case we have the same issue.